user_guide_src/source/testing/debugging.rst

```diff
@@ -72,16 +72,17 @@ Choosing What to Show
 
 CodeIgniter ships with several Collectors that, as the name implies, collect data to display on the toolbar. You
 can easily make your own to customize the toolbar. To determine which collectors are shown, again head over to
-the App configuration file::
-
-	public $toolbarCollectors = [
-		'CodeIgniter\Debug\Toolbar\Collectors\Timers',
-		'CodeIgniter\Debug\Toolbar\Collectors\Database',
-		'CodeIgniter\Debug\Toolbar\Collectors\Logs',
-		'CodeIgniter\Debug\Toolbar\Collectors\Views',
- 		'CodeIgniter\Debug\Toolbar\Collectors\Cache',
-		'CodeIgniter\Debug\Toolbar\Collectors\Files',
-		'CodeIgniter\Debug\Toolbar\Collectors\Routes',
+the **app/Config/Toolbar.php** configuration file::
+
+	public $collectors = [
```

Comment:
This should not have the `::class` part.

or `'\CodeIgniter\Debug\Toolbar\Collectors\Timers::class'` should be changed to `\CodeIgniter\Debug\Toolbar\Collectors\Timers::class`

Reply:
thanks. you is right.